packages/web/src/components/edit-track/EditTrackModal.tsx

```diff
@@ -79,12 +84,20 @@ const EditTrackModal = ({
       // Update the download json field based on the is_downloadable flag and the download conditions.
       // Note that this only needs to be done temporarily until the backend is updated to remove the download fields redundancy.
       // TODO: Remove this once the backend is updated to remove the download fields redundancy.
+      const is_downloadable = isLosslessDownloadsEnabled
```

Comment:
downloadable behavior was wrong if feature flag is false, which is what prod would have. fixed here